script/.contributors_aliases.json

```diff
@@ -169,5 +169,9 @@
   "ville.skytta@iki.fi": {
     "mails": ["ville.skytta@iki.fi", "ville.skytta@upcloud.com"],
     "name": "Ville Skyttä"
+  },
+  "adam.grant.hendry@gmail.com": {
+    "mails": ["adam.grant.hendry@gmail.com"],
+    "name": "Adam Hendry"
```

Comment:
No need to open another merge request for this imo.

Reply:
Let me know if you want me to take it out. I'll leave it in for now. I'm fine either way.